Taken from upstream.

From dca11eb872c96f86388890b3750e450cc2a68700 Mon Sep 17 00:00:00 2001
From: Samuel Thibault <samuel.thibault@ens-lyon.org>
Date: Mon, 21 Dec 2020 16:01:07 +0000
Subject: [PATCH] hurd: Fix getting VM_MIN/MAX_ADDRESS

gnu-nat.c was getting the inclusion of vm_param.h only by luck. We need
to explicitly include it to be sure to get the definitions of
VM_MIN/MAX_ADDRESS.

gdb/ChangeLog:

	* gnu-nat.c: Include <mach/vm_param.h>.
---
 gdb/gnu-nat.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/gdb/gnu-nat.c b/gdb/gnu-nat.c
index 1693ce0c43d..6f290be0d51 100644
--- a/gdb/gnu-nat.c
+++ b/gdb/gnu-nat.c
@@ -32,6 +32,7 @@ extern "C"
 #include <mach/message.h>
 #include <mach/notify.h>
 #include <mach/vm_attributes.h>
+#include <mach/vm_param.h>
 
 #include <hurd.h>
 #include <hurd/interrupt.h>
